### DISCUSSION TITLE
Collaboration

### DISCUSSION THREAD

**Discussion by wrthwhl** [foambubble/foam#1305] [General]
Hi all

I am really intrigued by foam. So much so that I would very much love to be able to use foam with my team at work. However, I have not really read too much about the collaborative use of foam. I would be very interested to learn about the following:

* Plans/intentions of foam to cater to teams
* Ideas on how foam could best be used by teams today as well as known limitations
* Experiences of others with the use of foam in teams

Many thanks in advance!
Marco

**Comment by riccardoferretti** [comment]
Can you tell me what type of team features are you thinking about?
Being based on VS Code, you can have collaborative editing, and with many people using it in combination with git it can work as a versioned documentation repo, and being markdown you can publish the content with various frameworks.
Not sure if those "qualify", or what you had in mind. Let me know, I am curious about it.

**Comment by wrthwhl** [comment]
I would say the main challenge for me is async collaboration, specifically commenting, but also things like tagging, notifications about updates, etc. We are currently using notion and being a virtual team we rely heavily on asynchronous communication. I have tried using footnotes but it gets quite messy very quickly. Another possibility I see is to use PRs to use github for collaboration features, but that too seems like it would be too much of an "abuse" of github for it to be really productive.
I also find the idea of a markdown-based kanban board very intriguing. There are a few extensions (e.g. [this one](https://marketplace.visualstudio.com/items?itemName=coddx.coddx-alpha)) but none of them seem to have any substantial support. I wonder what I'm missing because I don't really understand why this idea hasn't taken off more.

Another gap to something like notion would be access restrictions (e.g. being able to share certain documents/folders only with specific users). But I accept that foam is the wrong place to look for that.

When you mentioned collaborative editing you had the synchronous version with e.g. Live Share in mind, right? The fact that foam is based on git and markdown is definitely among the main reasons I love the idea of foam.

Are you aware of others who are using foam with teams?

**Comment by riccardoferretti** [comment]
I have heard of people using Foam for team collaboration, but mostly around maintaining/publishing documentation.
A lot of the features you are suggesting are interesting, but probably outside of the current scope of Foam.
Curious though to hear if other people have comments around it

**Comment by dstengle** [comment]
I've struggled with this a bit when I want to originate something in my knowledge base but then collaborate with someone for feedback and editing. I ended up using google docs to work with my wife on editing some content.

What I'd really like is a way to do collaborative editing on specific markdown files and have that synced with the git repo with a friendly non-technical interface. I think this is out of scope for foam and in the future I will be trying some other solution.

As far as a shared knowledge base, I think there are a lot of solutions more suited to that for technical teams and that a second-brain system is trying to accomplish different things in terms of organization and customization.
